### ISSUE TITLE
Upgrade ESLint config format

### ISSUE THREAD

**Issue by lemonmade** [lemonmade/quilt#584]
https://eslint.org/docs/latest/use/configure/migration-guide?utm_source=ESnextNews.com&utm_medium=Weekly+Newsletter&utm_campaign=2023-07-18

**Comment by cartogram** [comment]
Hey @lemonmade! I noticed you removed ESLint, are you just going with prettier and TypeScript? Does this have to do with the v9 alpha? Wondering what you are thinking...

**Comment by lemonmade** [comment]
Hey Matt, I hope you are doing well! Thanks for the reminder to close this issue — I did indeed decide to remove ESLint in favor of just using Prettier. For me, personally, Prettier gives 90% of the value in linting, and I was more often annoyed by ESLint rules than I was happy to have them around. It doesn’t help that there is this big change coming to ESLint config, and that there are a collection of tools focused on being "eslint, but faster".

I figured that any project that wants to use ESLint can continue to do so on top of the basic linting/ formatting provided by the template projects.
